sasl-lib/private/crypto.rkt

```diff
@@ -11,8 +11,16 @@
 (define-cpointer-type _EVP_MD)
 (define-cpointer-type _EVP_MD_CTX)
 
-(define-libcrypto EVP_MD_size (_fun _EVP_MD -> _int))
-(define-libcrypto EVP_MD_block_size (_fun _EVP_MD -> _int))
+(define-libcrypto EVP_MD_size (_fun _EVP_MD -> _int)
+  #:make-fail (lambda (name)
+                (lambda ()
+                  (get-ffi-obj 'EVP_MD_get_size libcrypto (_fun _EVP_MD -> _int)
+                               (make-not-available name)))))
+(define-libcrypto EVP_MD_block_size (_fun _EVP_MD -> _int)
```

Comment:
It looks like `EVP_MD_block_size` is never used, so I'll just go ahead and drop it.